src/content/chapters/03_word_types.mdx

```diff
@@ -0,0 +1,181 @@
+
+Just like the letter h, a medial pair can only appear between two vowels.
+
+### Roots
+
+Roots are the building blocks of the language and express meanings that would be
+too hard or long to convey otherwise. 
+
+There are two options for a root's structure.
+
+**Root Structure Option 1:**
+
+1. Begin the root with a non-sonorant consonant.
+1. Follow it with at least one vhowel.
+1. Two options:
+   1. Either follow it with a sonorant and any number of vhowels.
+   1. Or follow it with a medial pair and at least one vhowel.
+1. Optionally repeat step 3 as much as you'd like.
+1. End.
+
+Examples:
+
+- ber: property for "_ is Eberbanic/reflects Eberbanic culture".
+- ban: property for "_ is a language".
+- TODO INSERT TRANSITIVE:
+- TODO INSERT TRANSITIVE:
+
+**Root Structure Option 2:**
+
+1. Begin the root with an initial pair.
+1. End it with at least one vhowel.
+1. Optionally:
+   1. Either end it with a sonorant, and optionally go back to step 2.
+   1. Or follow it with a medial consonant pair and go back to step 2.
+
+Examples:
+
+- pcien: property for "_ is bread".
+- ckual: property for "_ is a planet".
+- TODO INSERT TRANSITIVE: 
+- gli: relation for "_ is happy that _ is true".
+
+### Compounds
+
+Compounds let you combine words to make a new word, though you cannot combine
+compounds.
+
+Compounds start with "e", "a", or "o" and can be written with/without spaces due
+to Eberban's self-segregating morphology.
+
+Examples:
+
+- "e" marks a two-word compound: `eberban` (property for "_ is the eberban
+language").
+- "a" marks a three-word compound: `acepobu` (property for "is a dictionary").
+- "o" marks a four-or-more compound. Currently, none exist.
+
+Because we don't know how long a four-or-more compound could be, they must be
+ended with `'o` or ` o`.
+
+### Borrowings
+
+Borrowings let you import foreign words or names. They are not present in the
+dictionary, but can appear in compounds combined with a root word.
+
+Borrowings begin with `u`. If the first letter of the borrowed word is `u` too,
+then the word-boundary-marker must be appended to separate it from the prefix.
+
+**Borrowing Structure:**
+
+1. Prefix the word with `u`, and append `'` if the first letter will be `u`.
+1. Begin it with one of the following:
+   1. A single letter (yes this includes h)
+   1. An initial or medial pair
+   1. A consonant triplet
+1. Optionally continue alternating between vhowels and consonant pairs/triplets.
+1. End it:
+   1. Either with a vowel
+   2. Or a vowel and a single consonant.
+
+C1C2C3 is a **consonant triplet** if:
+
+- Either C1C2 is a medial pair and C2C3 is an initial pair.
+- Or C1 is a sonorant and C2C3 is an initial pair.
+
+Examples:
+
+- eufransestan: property for "_ is France".
+- euinglicban: property for "_ is the English language".
+- za u'unicef: property for "_ is named UNICEF".
+
+Borrowings are terminated with the particle `be`, but this is only required when
```

Comment:
Maybe give more details about that. 2 consecutive borrowings not separated with `be` merge together:
- za udjon ufordo = E is named John Ford
- za udjon be ufordo = E is named John and is "a ford" (maybe the car brand?).